data/35e/wizards_of_the_coast/supplement/fiend_folio/ff_classes.lst

```diff
@@ -0,0 +1,99 @@
+3	ABILITY:Fiend of Blasphemy Class Feature|AUTOMATIC|Fiend of Blasphemy ~ Sponsor Worshiper
+4	ABILITY:Fiend of Blasphemy Class Feature|AUTOMATIC|Fiend of Blasphemy ~ Scry on Cultist
+4	ABILITY:Fiend of Blasphemy Class Feature|AUTOMATIC|Fiend of Blasphemy ~ Torture Cultist
+5	ABILITY:Fiend of Blasphemy Class Feature|AUTOMATIC|Fiend of Blasphemy ~ Mind Shielding
+6	ABILITY:Fiend of Blasphemy Class Feature|AUTOMATIC|Fiend of Blasphemy ~ Kill Cultist
+#
+# spells
+#
+1	CAST:0
+2	CAST:1,0
+3	CAST:1,1,0
+4	CAST:1,1,1,0
+5	CAST:2,1,1,1,0
+6	CAST:2,2,1,1,1
+
+
+#Fiend of Corruption
+CLASS:Fiend of Corruption	HD:6	FACT:Abb|FoC	TYPE:PC.Prestige	MAXLEVEL:6
+CLASS:Fiend of Corruption	PRECHECK:1,Will=7
+CLASS:Fiend of Corruption	PRESKILL:1,Bluff=10	PRESKILL:1,Disguise=10	PRESKILL:1,Sense Motive=10
+CLASS:Fiend of Corruption	PRETEXT:Must be able to use Charm Person or Charm Monster as spell or spell-like ability
+CLASS:Fiend of Corruption	PRERACE:1,RACETYPE=Outsider		PRERACE:1,RACESUBTYPE=Evil
+CLASS:Fiend of Corruption	STARTSKILLPTS:2		CSKILL:Bluff|Diplomacy|Diguise|Knowledge (Religion)|Listen|Search|Sense Motive|Spot
+#CLASS:Fiend of Corruption	
+#
+CLASS:Fiend of Corruption	DEFINE:FoCLVL|0		BONUS:VAR|FoCLVL|CL
+#
+# levels
+#1	ABILITY:Fiend of Corruption Class Feature|AUTOMATIC|Fiend of Corruption ~ 
+1	ABILITY:Fiend of Corruption Class Feature|AUTOMATIC|Fiend of Corruption ~ Alternate Form
+1	ABILITY:Fiend of Corruption Class Feature|AUTOMATIC|Fiend of Corruption ~ Fiend's Favor
+2	ABILITY:Fiend of Corruption Class Feature|AUTOMATIC|Fiend of Corruption ~ Suggestion
+2	ABILITY:Fiend of Corruption Class Feature|AUTOMATIC|Fiend of Corruption ~ Mind Shielding
+3	ABILITY:Fiend of Corruption Class Feature|AUTOMATIC|Fiend of Corruption ~ Mark of Iniquity
+4	ABILITY:Fiend of Corruption Class Feature|AUTOMATIC|Fiend of Corruption ~ Fiendish Graft
+4	ABILITY:Fiend of Corruption Class Feature|AUTOMATIC|Fiend of Corruption ~ Major Creation
+5	ABILITY:Fiend of Corruption Class Feature|AUTOMATIC|Fiend of Corruption ~ Geas
+5	ABILITY:Fiend of Corruption Class Feature|AUTOMATIC|Fiend of Corruption ~ Temptation
+6	ABILITY:Fiend of Corruption Class Feature|AUTOMATIC|Fiend of Corruption ~ Grant Wish
+6	ABILITY:Fiend of Corruption Class Feature|AUTOMATIC|Fiend of Corruption ~ Soul Bargain
+
+#Fiend of Possession
+CLASS:Fiend of Possession	HD:6	FACT:Abb|FoP	TYPE:PC.Prestige	MAXLEVEL:6
+CLASS:Fiend of Possession	PRECHECK:1,Will=5
+CLASS:Fiend of Possession	PRESKILL:1,Hide=6	PRESKILL:1,Knowledge (Arcana)=6
+#CLASS:Fiend of Possession	
+CLASS:Fiend of Possession	PRERACE:1,RACETYPE=Outsider		PRERACE:1,RACESUBTYPE=Evil
+CLASS:Fiend of Possession	STARTSKILLPTS:2		CSKILL:Bluff|Diplomacy|Diguise|Hide|Listen|Search|Sense Motive|Spellcraft|Spot
+#CLASS:Fiend of Possession
+#
+CLASS:Fiend of Possession	DEFINE:FoPLVL|0		BONUS:VAR|FoPLVL|CL	DEFINE:FoPPossessionDC|0	BONUS:VAR|FoPPossessionDC|(10+FoPLVL+CHA)
```

Comment:
Standard convention is DEFINE and BONUS go in the class feature. Think of the class feature as a container, everything the belongs to the container should be found inside. If you look at the classes I have updated you would see very little remains in the class itself. From a practical stand point, it prevent needless work for something that the class does not even possess (pun intended).

Another convention for clean reading, '###Block' separates the different sections. This is actually required if we wish to maintain the look when the prettylst program is used.